### ISSUE TITLE
Move DataExtractors here?

### ISSUE THREAD

**Issue by casaout** [sealuzh/ApplicationToActivityMapper#2]
Does it make sense to move the [data extractors](https://github.com/sealuzh/PersonalAnalytics/tree/dev-am/src/client/AM.PA.MonitoringTool/Shared/Data/Extractors) here?

**Comment by RaphaelRo** [comment]
I am not familiar with the Extractors but by the first look, it seems they are very specific to the database implementation? So imho not.

**Comment by casaout** [comment]
Sorry, I need to provide a bit more context ;) They basically take a window title, and extract things like File-Names, Website-Names, (sometimes) even People we collaborate with.

How we consume them is specific to the db; but we/I could extract the mapper. (Just yesterday, I had a conversation with someone, who is interested in using that part).

**Comment by RaphaelRo** [comment]
Oh thanks for the clarification. Yes - it makes sense then to put it into the package (assuming we can untangle it from the db without too much effort).

**Comment by casaout** [comment]
Yeah, I think we can. But that's probably something I should do 😯
